volumes/ecocounter/tables/counts.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE ecocounter.counts (
+    flow_id numeric REFERENCES ecocounter.locations (flow_id),
```

Comment:
Do you find yourself joining to the flows table to get the site_id a lot? Wondering if it would be helpful to include site_id here to reduce the need of joining tablels

Reply:
Not sure it makes much difference (in terms of speed) as the sites and flows tables will always be very small. I think I'd actually kind of rather have the extra join just to make clear how the concepts relate (i.e. that each site has 1+ flows)